### PR TITLE
Adds the T-42 LMG for PFCs

### PR DESCRIPTION
**Pull request by Kuro-ai**
## About The Pull Request

Adds the T-42 Light machine gun for PFCs

## Why It's Good For The Game

It's the second weapon in the T series following the DMR,adds the DAKKA DAKKA we currently lack

## Changelog
:cl:
add: Adds T-42 LMG for PFCs
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -5. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Pariah919** [comment]
finally

File: code/datums/gamemodes/distress.dm [hunk 1]
```diff
@@ -318,6 +318,7 @@
 						/obj/item/ammo_magazine/rifle/sx16_buckshot = round(scale * 10),
 						/obj/item/ammo_magazine/rifle/sx16_flechette = round(scale * 10),
 						/obj/item/ammo_magazine/rifle/sx16_slug = round(scale * 10),
+						/obj/item/ammo_magazine/standard_lmg = round(scale * 15),
```
Comment (Rohesie) [inline code/datums/gamemodes/distress.dm]: The gun is a child of rifle. The magazine should be as well.

File: code/modules/projectiles/updated_projectiles/guns/rifles.dm [hunk 2]
```diff
@@ -343,7 +343,45 @@
...
+						/obj/item/attachable/magnetic_harness,
+						/obj/item/attachable/scope)
+
+	flags_gun_features = GUN_AUTO_EJECTOR|GUN_CAN_POINTBLANK|GUN_AMMO_COUNTER|GUN_WIELDED_FIRING_ONLY|GUN_LOAD_INTO_CHAMBER
+	gun_firemode_list = list(GUN_FIREMODE_SEMIAUTO, GUN_FIREMODE_AUTOMATIC)
+	starting_attachment_types = list(/obj/item/attachable/stock/dmr, /obj/item/attachable/t42barrel)
+	gun_skill_category = GUN_SKILL_HEAVY_WEAPONS
+	attachable_offset = list("muzzle_x" = 33, "muzzle_y" = 17,"rail_x" = 10, "rail_y" = 20, "under_x" = 24, "under_y" = 13, "stock_x" = 12, "stock_y" = 12)
+	fire_delay = 0.2 SECONDS
+	burst_amount = 1
```
Comment (Rohesie) [inline code/modules/projectiles/updated_projectiles/guns/rifles.dm]: If it doesn't accept a BFA and you don't want it to burst, set this to zero.

File: code/modules/projectiles/updated_projectiles/magazines/rifles.dm [hunk 3]
```diff
@@ -103,6 +103,18 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	max_rounds = 200
 	gun_type = /obj/item/weapon/gun/rifle/lmg
+//-------------------------------------------------------
+//T42 LIGHT MACHINE GUN
+
+/obj/item/ammo_magazine/standard_lmg
+	name = "\improper T-42 ammo box (10x24mm)"
+	desc = "A semi-rectangular box of rounds for the M41AE2 Heavy Pulse Rifle."
```
Comment (Alterist) [inline code/modules/projectiles/updated_projectiles/magazines/rifles.dm]: wrong description

**Pull request closed without merging**